radio_beam/beam.py

```diff
@@ -27,6 +27,16 @@ def _to_area(major,minor):
                u.arcmin: "'"}
 
 
+def _set_default_unit(type_str, value, unit, equiv_unit=u.deg):
```

Comment:
I'm not sure why it's called `_set_default_unit` if it's returning the quantity.  Maybe `_with_default_unit`?